### ISSUE TITLE
TypeError: Cannot read property 'hide' of undefined

### ISSUE THREAD

**Issue by probonopd** [harshjv/nifty#1]
`nifty-1.0.2-x86_64.AppImage` throws on Ubuntu 16.04:

```
Uncaught Exception:
TypeError: Cannot read property 'hide' of undefined
    at App.app.on (/tmp/.mount_tAEEIM/usr/bin/resources/app.asar/index.js:14:11)
    at emitTwo (events.js:111:20)
    at App.emit (events.js:191:7)
```

**Comment by harshjv** [comment]
Thanks for reporting the issue. This is because `app.dock` is undefined on platforms other than macOS.

**Comment by harshjv** [comment]
This is fixed in v1.0.3
